tests/ranger/container/test_bookmarks.py

```diff
@@ -56,3 +56,20 @@ def crash():
         secondstore.update_if_outdated()
     secondstore.update = origupdate
     secondstore.update_if_outdated()
+
+
+def test_bookmark_symlink(tmpdir):
+    # Initialize plain file and symlink paths
+    bookmarkfile_link = tmpdir.join("bookmarkfile")
+    bookmarkfile_orig = tmpdir.join("bookmarkfile.org")
+
+    # Create symlink pointing towards the original plain file.
+    os.symlink(str(bookmarkfile_orig), str(bookmarkfile_link))
+
+    # Initialize the bookmark file and save the file.
+    bmstore = Bookmarks(str(bookmarkfile_link))
+    bmstore.save()
+
+    # Once saved, the bookmark file should still be a symlink pointing towards the plain file.
```

Comment:
As a formality, I'd check the symlink target path too, not only it being a symlink, since the comment suggests it is checked.

Reply:
The check happens after the bookmark file is saved, since the `bookmarkfile_orig` does not  exist until `bmstore.save()` has been called.

Reply:
Good point.